python/lsst/ts/xml/enums/MTDome.py

```diff
@@ -168,6 +169,15 @@ class RadLockingPinState(enum.IntEnum):
     DISENGAGED = 2
 
 
+class PowerManagementMode(enum.IntEnum):
+    """Power management mode."""
+
+    NO_POWER_MANAGEMENT = 0
```

Comment:
I think you should probably start at 1 instead of 0. Note that, by default, enumerations start at 1. So the way you defined it in the xml `NO_POWER_MANAGEMENT` is supposed to be 1. 

If you really want this to start at zero (which I advise against), you will need to explicitly set the values in the xml, e.g.;

```
  <Enumeration>
    PowerManagementMode_NO_POWER_MANAGEMENT=0,
    PowerManagementMode_OPERATIONS=1,
    PowerManagementMode_EMERGENCY=2,
    PowerManagementMode_MAINTENANCE=3
  </Enumeration>
```

Reply:
Done. Note that I renumbered the RadLockingPinState enum values as well since they too started at 0.